source/tutorials/schematics.md

```diff
@@ -26,6 +27,25 @@ Once you have scanned a structure, you can use the build tool to have your build
 
 ![Schematic](../../assets/images/tutorial/schematic.png)
 
+
+## Converting Files
+
+If you have old schematics that have the .nbt file name, you can automatically convert them to the new .blueprint name without rescanning the. The newer updates will not allow .nbt files to be used so you have to convert them first.
+
+To Convert NBT to .Blueprint
+
+1. Open up your structurize folder eg: C:\Users\YourName\Documents\Curse\Minecraft\Instances\Myserver\structurize
+2. Create a folder called "updater"
+3. Create a folder under updater called "input"
+4. Move NBT files to the sturcturize\updater\input folder
+5. Enter an SP world in that instance 
+6. Type in the command /updateschematics
```

Comment:
The  command is `/structurize updateschematics`

Reply:
Fixed